reinforcement_learning/actor_critic.py

```diff
@@ -54,29 +56,28 @@ def forward(self, x):
 def select_action(state):
     state = torch.from_numpy(state).float().unsqueeze(0)
     probs, state_value = model(Variable(state))
-    action = probs.multinomial()
-    model.saved_actions.append(SavedAction(action, state_value))
+    m = Multinomial(probs)
+    action = m.sample()
+    model.saved_actions.append(SavedAction(m.log_prob(action), state_value))
```

Comment:
Cool. Looks good. I guess we can unwrap `state_value`, since we wont backprop through it?

Reply:
We will backprop through it - the critic needs updating (via the value loss). We just don't want the critic to be accidentally updated by the policy loss, so the policy loss uses the unwrapped value.